hudi-examples/hudi-examples-spark/src/main/java/org/apache/hudi/examples/spark/HoodieSparkBootstrapExample.java

```diff
@@ -64,7 +64,6 @@ public static void main(String[] args) throws Exception {
             .option(DataSourceWriteOptions.PRECOMBINE_FIELD().key(), preCombineField)
             .option(HoodieTableConfig.BASE_FILE_FORMAT.key(), HoodieFileFormat.ORC.name())
             .option(HoodieBootstrapConfig.BASE_PATH.key(), basePath)
-            .option(HoodieBootstrapConfig.KEYGEN_CLASS_NAME.key(), NonpartitionedKeyGenerator.class.getCanonicalName())
```

Comment:
shouldn't we replace w/ data source key gen class config ?

Reply:
Fixed.